parsec/core/win_registry.py

```diff
@@ -173,22 +175,23 @@ def del_parsec_drive_icon(letter: str):
 
 
 @contextmanager
-def parsec_drive_icon_context(letter):
+def parsec_drive_icon_context(letter: str) -> Iterator[None]:
     # Winreg is not available for some reasons
     if sys.platform != "win32" or not try_winreg():
         yield
         return
 
     # Safe context for removing the key after usage
-    with importlib.resources.files(resources).joinpath(DRIVE_ICON_NAME) as drive_icon_path:
-        set_parsec_drive_icon(letter, drive_icon_path)
-        try:
-            yield
-        finally:
-            del_parsec_drive_icon(letter)
+    # mypy: This cast is safe because Traversable is a subset if pathlib.Path
```

Comment:
Left over comment ?

Reply:
Yes, I forgot to remove them but note that on Windows a `# type: ignore[return]` is needed here. However mypy complains about unused type ignore on other platforms :(